src/onegov/agency/views/agencies.py

```diff
@@ -262,14 +263,17 @@ def cache_headers(response):
     form=Form
 )
 def create_root_pdf(self, request, form):
+    orga = request.session.query(Organisation).one()
```

Comment:
I'd say the canonical variable name for an organisation is `org`.

Reply:
Also, you don't need to query the organisation anywhere. The organisation is always cached on request.app.org (guaranteed to be up to date, but cached).

Reply:
changed...